_inc/lib/admin-pages/class.jetpack-react-page.php

```diff
@@ -420,3 +420,23 @@ function jetpack_current_user_data() {
 
 	return $current_user_data;
 }
+
+/**
+ * Set the admin language, based on user language.
+ *
+ * @since 4.5.0
+ *
+ * @return string
+ *
+ * @todo Remove this function when WordPress 4.8 is released
+ * and replace `jetpack_get_locale()` in this file with `get_user_locale()`.
+ */
+function jetpack_get_locale() {
```

Comment:
Make sense. It actually fallback to `get_locale` which is equal to `get_user_locale` in 4.6 or lower